setup.cfg

```diff
@@ -78,15 +78,14 @@ datasets =
     h5py>=2.6,<4
     # laspy 2+ required for laspy.read
     laspy>=2,<3
-    # open3d 0.11.2+ required to avoid GLFW error:
-    # https://github.com/isl-org/Open3D/issues/1550
-    open3d>=0.11.2,<0.15;python_version<'3.10'
     # opencv-python 3.4.2.17 is oldest buildable version on PyPI
     opencv-python>=3.4.2.17,<5
     # pandas 0.23.2+ required for Python 3.7 wheels
     pandas>=0.23.2,<2
     # pycocotools 2.0.0 is oldest version on PyPI
     pycocotools>=2,<3
+    # pyvista required for point cloud plotting
+    pyvista
```

Comment:
Last step is to determine the minimum supported version of pyvista. @banesullivan any guess as to the minimum version of pyvista that provides the features we use? If not we'll just do a binary search on all versions until we find where the tests start to fail.

Reply:
I get segmentation faults even when using xvfb when these tests run so I'll need to move over to my mac and run them. This should be fun

Reply:
> any guess as to the minimum version of pyvista that provides the features we use?

Hm... do you *need* to provide a min version? The features used here have been around for quite some time. I'd recommend not setting the min version as far back as you can, but arbitrarily choose a more recent version for stability. Again, IMO, I would leave the version unconstrained in the `setup.cfg` but if you must, I'd target v0.33 or v0.34 (we dropped Python3.6 support in v0.34)

Reply:
I would like to provide a minimum version just to be explicit. For some packages like OpenCV it's the minimum version released on PyPI because all versions worked. For others it's the minimum version that supports Python 3.7 (the oldest we support). The goal is that the widest possible range is supported. We have unit tests that run our unit tests with the oldest supported version so we can ensure that it is correct and users are unlikely to encounter an invalid environment.

Reply:
The real versioning issues might come from VTK... You'll need to be careful if installing an older version of PyVista with newer versions of VTK. More recent versions of PyVista do a better job of targeting all versions of VTK whereas older versions will have conflicts on import with a recent version of VTK. 

This can get quite complicated quite quickly.

I'd recommend choosing a recent min version of PyVista (e.g., 0.34) and limiting VTK to 9.x.x (though it is a really, really bad idea to set a version constraint on VTK in the setup.cfg)

Reply:
> > I get segmentation faults even when using xvfb when these tests run so I'll need to move over to my mac and run them. This should be fun
> 
> Are you doing this locally? You only need xvfb in headless environments

I'm working out of an ec2 instance so I was trying to it out

Reply:
Probably quicker to do a binary search on versions using GitHub Actions than it is to test locally or on ec2.

We probably won't be able to control which version of VTK gets installed since it's a non-Python dependency. I say we go with the oldest version of PyVista that works in CI without messing with any system packages. That's what we do for all other dependencies.

Reply:
>We probably won't be able to control which version of VTK gets installed since it's a non-Python dependency

Nope, it's a Python dep! Its wheels are on PyPI, and you can easily control what version get's installed with pip. Pinning in the setup.cfg can be a bad idea because a lot of folks use VTK from different sources (e.g. ParaView, built from source, etc and the version may not resolve)

Reply:
Gotcha. We probably won't pin vtk in `setup.cfg`, although we may pin in `requirements/min.old` since that's what's used in CI.